data/commodities.txt

```diff
@@ -416,9 +531,12 @@ trade
 		"first aid kits"
 		"forceps"
 		"gauze pads"
+		"gloves"
+		"hairnets"
 		"hospital beds"
 		"hydrogen peroxide"
 		"intravenous fluids"
+		"medical goods"
```

Comment:
Same as with `food` sort of. The commodity lists are designed to make things more specific and diverse, but this is no more specific than the category name.